### DISCUSSION TITLE
Github pages not updating (newest commit not reflecting on website)

### DISCUSSION THREAD

**Discussion by tiyalla** [community/community#22329] [Other Feature Feedback, Questions, & Ideas]
<p>Can anyone please help me understand why it is taking so long for my changes that are deployed to be reflected on my personal website that is hosted on github pages? It has been over two hours now and I don’t see my new blog post yet. I have cleared my caches and nothing. Has anyone else faced a similar situation?</p>

**Comment by tiyalla** [comment, marked as answer]
<p>This solved my issue. I needed to add git submodules to the public dir</p>
<p><a href="https://github.com/whipperstacker/blog/blob/master/content/post/deploying-a-hugo-site-to-github-pages.md" rel="nofollow noopener">https://github.com/whipperstacker/blog/blob/master/content/post/deploying-a-hugo-site-to-github-pages.md </a></p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/tiyalla">@tiyalla</a>, Thanks for coming back to share what worked for you!</p>
